File: android/SharedPreferences_ex1/app/src/main/java/com/mastermovilesua/persistencia/preferencias/sharedpreferences1/Base64Java/Base64Cipher.java
package com.mastermovilesua.persistencia.preferencias.sharedpreferences1.Base64Java;

import android.util.Base64;

public class Base64Cipher {
    public static String encryptData(String data) {
        return Base64.encodeToString(data.getBytes(), Base64.DEFAULT);
    }

    public static String decryptData(String data) {
        return new String(Base64.decode(data.getBytes(), Base64.DEFAULT));
    }
}

File: android/SharedPreferences 1/SharedPreferences_ex1/app/src/main/java/com/mastermovilesua/persistencia/preferencias/sharedpreferences1/Base64Kotlin/Base64Cipher.kt
package com.mastermovilesua.persistencia.preferencias.sharedpreferences1.Base64Kotlin

import android.util.Base64

object Base64Cipher {
    fun encryptData(data: String): String {
        return Base64.encodeToString(data?.toByteArray(), Base64.DEFAULT)
    }

    fun decryptData(data: String): String {
        return String(Base64.decode(data?.toByteArray(), Base64.DEFAULT))
    }
}